generators/client/templates/angular/src/test/javascript/_karma.conf.js

```diff
@@ -1,88 +1,128 @@
-// Karma configuration
-// http://karma-runner.github.io/0.13/config/configuration-file.html
+'use strict';
 
-var sourcePreprocessors = ['coverage'];
-
-function isDebug() {
-    return process.argv.indexOf('--debug') >= 0;
-}
-
-if (isDebug()) {
-    // Disable JS minification if Karma is run with debug option.
-    sourcePreprocessors = [];
-}
+const path = require('path');
+const webpack = require('webpack');
+const WATCH = process.argv.indexOf('--watch') > -1;
+const LoaderOptionsPlugin = require("webpack/lib/LoaderOptionsPlugin");
 
 module.exports = function (config) {
     config.set({
-        // base path, that will be used to resolve files and exclude
-        basePath: '<%= TEST_SRC_DIR %>'.replace(/[^/]+/g, '..'),
 
-        // testing framework to use (jasmine/mocha/qunit/...)
+        // base path that will be used to resolve all patterns (eg. files, exclude)
+        basePath: './',
+
+        // frameworks to use
+        // available frameworks: https://npmjs.org/browse/keyword/karma-adapter
         frameworks: ['jasmine'],
 
         // list of files / patterns to load in the browser
         files: [
-            // bower:js
-            // endbower
-            '<%= MAIN_SRC_DIR %>app/app.module.js',
-            '<%= MAIN_SRC_DIR %>app/app.state.js',
-            '<%= MAIN_SRC_DIR %>app/app.constants.js',
-            '<%= MAIN_SRC_DIR %>app/**/*.+(js|html)',
-            '<%= TEST_SRC_DIR %>spec/helpers/module.js',
-            '<%= TEST_SRC_DIR %>spec/helpers/httpBackend.js',
-            '<%= TEST_SRC_DIR %>**/!(karma.conf<% if (testFrameworks.indexOf("protractor") > -1) { %>|protractor.conf<% } %>).js'
+            'spec/entry.ts'
         ],
 
 
-        // list of files / patterns to exclude
-        exclude: [<% if (protractorTests) { %>'<%= TEST_SRC_DIR %>e2e/**'<% } %>],
+        // list of files to exclude
+        exclude: [<% if (protractorTests) { %>'e2e/**'<% } %>],
```

Comment:
I don't think this is needed because `spec/entry.ts` gets the list of files from its current directory. However, it probably doesn't hurt to have it in there.